libretroshare/src/pqi/pqissludp.cc

```diff
@@ -430,30 +414,6 @@ int pqissludp::net_internal_SSL_set_fd(SSL *ssl, int fd)
 	return 1;
 }
 
-int pqissludp::net_internal_fcntl_nonblock(int /*fd*/)
```

Comment:
There's quite a hierarchy of classes above and below pqissludp. Sometimes you find functions that do nothing (like here) but prevent calling overloaded methods in subclasses. Did you check that?